bin/picca_fast_metal_dmat.py

```diff
@@ -0,0 +1,567 @@
+#!/usr/bin/env python
+"""Compute the auto and cross-correlation of delta fields for a list of IGM
+absorption.
+
+This module follow the procedure described in sections 4.3 of du Mas des
+Bourboux et al. 2020 (In prep) to compute the distortion matrix
```

Comment:
Isn't  this de DR16 paper? If so, then it is no longer in Prep.

Reply:
I changed the description